android/build.gradle

```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "28.0.3"
-        minSdkVersion = 16
+        minSdkVersion = 21
```

Comment:
@armpitude This is crucial. I had to upgrade this because it was throwing some errors in the crypto repo. Make sure this doesn't break anything for you on Android. I think it should be fine, but let me know